### DISCUSSION TITLE
Separate conversations from other events on the PR summary page

### DISCUSSION THREAD

**Discussion by akwodkiewicz** [community/community#52610] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Please take a look at the following screenshot

![image](https://user-images.githubusercontent.com/22861194/231418126-66e4e205-4ac6-4279-ab77-cda944bc38db.png)

The PR contains various "add/remove label" actions, various commits being pushed, multiple deployments onto various environments and... unresolved conversations.

It's the conversations with other contributors that move the development forward. It would be best if the standalone comments and the review threads were not **hidden** like that in the plethora of other events.

Can we have the conversations shown on some other part of the summary page? Both unresolved and resolved (including standalone comments)?

**Comment by PatrykKuniczak** [comment]
That's great idea, i think about it, to split this section, into 2 resolved and unresolved as you write.

That's can be very helpful.

**Comment by 403-html** [comment]
bump as it might be saving tons of time

**Comment by quezak** [comment]
Besides unresolved conversations, it would also be nice to have review approvals and rejections visible right away, at least those that were not dismissed.

**Comment by beardedramp** [comment]
Any update on that? This is so annoying.
